.github/workflows/acceptance-tests.yml

```diff
@@ -103,6 +106,28 @@ jobs:
             - 'mongodbatlas/resource_mongodbatlas_team*.go'
             - 'mongodbatlas/resource_mongodbatlas_third_party_integration*.go'     
 
+  fetch-sts-assume-role-creds:
+    runs-on: ubuntu-latest
+    outputs:
```

Comment:
don we need something like this?
 needs: [ change-detection ]
    if: ${{ needs.change-detection.outputs.assume-role == 'true'